VideoLocker/src/test/java/org/edx/mobile/test/http/ApiTests.java

```diff
@@ -55,10 +58,9 @@ public void setUp() throws Exception {
         super.setUp();
     }
 
+    @Ignore
```

Comment:
You should add a `// TODO: Debug and fix test failure` along with the ignored annotations.